bitnami/grafana-loki/Chart.yaml

```
@@ -56,4 +56,4 @@ maintainers:
 name: grafana-loki
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/grafana-loki
-version: 4.7.10
+version: 4.7.11
```

According to semver.org, we should bump a minor version here:

```suggestion
version: 4.8.0
```